controllers/activemqartemis_controller_test.go

```diff
@@ -9244,6 +9244,54 @@ var _ = Describe("artemis controller", func() {
 			Expect(k8sClient.Delete(ctx, &crd)).To(Succeed())
 		})
 
+		It("-bp suffix secret broker-n support", Label("broker-n-bp-secret"), func() {
+
+			ctx := context.Background()
+
+			_, bpSecret1 := DeploySecret(defaultNamespace, func(candidate *corev1.Secret) {
+				candidate.Name = "config-1-bp"
+				candidate.StringData = map[string]string{
+					"journal1.properties":       `journalFileSize=12345`,
+					"broker-0.first.properties": `globalMaxSize=512m`,
+				}
+			})
+
+			_, bpSecret2 := DeploySecret(defaultNamespace, func(candidate *corev1.Secret) {
+				candidate.Name = "config-2-bp"
+				candidate.StringData = map[string]string{
+					"journal2.properties":        `journalFileSize=12345`,
+					"broker-0.second.properties": `globalMaxSize=512m`,
```

Comment:
it may make sense to use different values for first and second, so they can be differenciated later

Reply:
+1